-------------------- Log Start | Mon Jan 20 15:13:20 2025 --------------------
platform: Linux-6.1.0-30-amd64-x86_64-with-glibc2.36
data_path: /home/secretjameser/Ender3Pro_data
is_default_data_path: False
config_file: /home/secretjameser/Ender3Pro_data/config/moonraker.conf
backup_config: /home/secretjameser/Ender3Pro_data/config/.moonraker.conf.bkp
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: f5a595eb585d48de9e462abc9c33a19b
unix_socket_path: /home/secretjameser/Ender3Pro_data/comms/moonraker.sock
software_version: v0.9.3-16-g72ed175
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/secretjameser/Ender3Pro_data/logs/moonraker.log
python_version: 3.11.2 (main, Nov 30 2024, 21:22:50) [GCC 12.2.0]
launch_args: /home/secretjameser/moonraker-env/bin/python /home/secretjameser/moonraker/moonraker/moonraker.py -d /home/secretjameser/Ender3Pro_data
msgspec_enabled: False
uvloop_enabled: False

#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
klippy_uds_address = /home/secretjameser/Ender3Pro_data/comms/klippy.sock

[authorization]
trusted_clients = 
	10.0.0.0/8
	127.0.0.0/8
	169.254.0.0/16
	172.16.0.0/12
	192.168.0.0/16
	FE80::/10
	::1/128
cors_domains = 
	*.lan
	*.local
	*://localhost
	*://localhost:*
	*://my.mainsail.xyz
	*://app.fluidd.xyz

[octoprint_compat]

[history]

[update_manager]
channel = dev
refresh_interval = 168

[update_manager fluidd-config]
type = git_repo
primary_branch = master
path = ~/fluidd-config
origin = https://github.com/fluidd-core/fluidd-config.git
managed_services = klipper

[update_manager fluidd]
type = web
channel = stable
repo = fluidd-core/fluidd
path = ~/fluidd

#################################################################
All Configuration Files:
/home/secretjameser/Ender3Pro_data/config/moonraker.conf
#################################################################
Loading Sqlite database provider. Sqlite Version: 3.40.1
Unsafe Shutdown Count: 0

System Info:

***python***
  version: (3, 11, 2, 'final', 0)
  version_string: 3.11.2 (main, Nov 30 2024, 21:22:50) [GCC 12.2.0]

***cpu_info***
  cpu_count: 4
  bits: 64bit
  processor: x86_64
  cpu_desc: Intel(R) Core(TM) i5 CPU       M 460  @ 2.53GHz
  serial_number: 
  hardware_desc: 
  model: 
  total_memory: 5859492
  memory_units: kB

***sd_info***

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.1.0-30-amd64

***virtualization***
  virt_type: none
  virt_identifier: none

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
Module libcamera unavailble, import failed

Systemd unit moonraker-Ender3Pro.service:
unit_name: moonraker-Ender3Pro.service
is_default: False
manager: systemd
Properties:
**Requires=['-.mount', 'sysinit.target', 'system.slice', 'network-online.target']
**After=['basic.target', 'sysinit.target', '-.mount', 'system.slice', 'network-online.target', 'systemd-journald.socket']
**SupplementaryGroups=['moonraker-admin']
**EnvironmentFiles=/home/secretjameser/Ender3Pro_data/systemd/moonraker.env
**ExecStart=/home/secretjameser/moonraker-env/bin/python $MOONRAKER_ARGS
**WorkingDirectory=/home/secretjameser/moonraker
**FragmentPath=/etc/systemd/system/moonraker-Ender3Pro.service
**Description=API Server for Klipper SV1 Ender3Pro
**User=secretjameser

Systemd unit klipper-Ender3Pro.service:
unit_name: klipper-Ender3Pro.service
is_default: False
manager: systemd
Properties:
**Requires=['system.slice', 'sysinit.target', '-.mount']
**After=['-.mount', 'network-online.target', 'sysinit.target', 'systemd-journald.socket', 'basic.target', 'system.slice']
**SupplementaryGroups=[]
**EnvironmentFiles=/home/secretjameser/Ender3Pro_data/systemd/klipper.env
**ExecStart=/home/secretjameser/klippy-env/bin/python $KLIPPER_ARGS
**WorkingDirectory=/home/secretjameser/klipper
**FragmentPath=/etc/systemd/system/klipper-Ender3Pro.service
**Description=Klipper 3D Printer Firmware SV1
**User=secretjameser
Git Repo moonraker: No warnings detected
Git Repo klipper: No warnings detected
Git Repo fluidd-config: No warnings detected
Klipper Version: v0.12.0-410-gcf3b0475d
2025-01-20 10:13:20,812 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.1.14
2025-01-20 10:13:20,860 [application.py:log_request()] - 101 GET /websocket?token=VMMHJYYD7CDAO5ALZVPX73YIZS5UUP5B (192.168.1.14) [_TRUSTED_USER_] 0.83ms
2025-01-20 10:13:20,860 [websockets.py:open()] - Websocket Opened: ID: 140519400346000, Proxied: False, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36 OPR/114.0.0.0 (Edition std-1), Host Name: 192.168.1.38
2025-01-20 10:13:20,872 [websockets.py:_handle_identify()] - Websocket 140519400346000 Client Identified - Name: fluidd, Version: 1.31.4-f283a37, Type: web
2025-01-20 10:13:20,872 [file_manager.py:get_file_list()] - Updating File List <config>...
2025-01-20 10:14:15,032 [application.py:log_request()] - 101 GET /websocket?token=LVGKN67RXMM2V6WUZFRS2CFCPNNHCCAM (192.168.1.14) [_TRUSTED_USER_] 0.90ms
2025-01-20 10:14:15,032 [websockets.py:open()] - Websocket Opened: ID: 140519410676752, Proxied: False, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36 OPR/114.0.0.0 (Edition std-1), Host Name: 192.168.1.38
2025-01-20 10:14:15,042 [websockets.py:_handle_identify()] - Websocket 140519410676752 Client Identified - Name: fluidd, Version: 1.31.4-f283a37, Type: web
2025-01-20 10:14:15,042 [file_manager.py:get_file_list()] - Updating File List <config>...
2025-01-20 10:14:35,384 [websockets.py:on_close()] - Websocket Closed: ID: 140519410676752 Close Code: None, Close Reason: None, Pong Time Elapsed: 0.34
2025-01-20 10:15:58,226 [application.py:prepare()] - Upload Request Received from 192.168.1.14
Content-Type: 
2025-01-20 10:15:58,235 [application.py:prepare()] - Upload Request Received from 192.168.1.14
Content-Type: multipart/form-data; boundary=----WebKitFormBoundaryP2gDY56SeTQFqWxG
2025-01-20 10:15:58,246 [application.py:post()] - Processing Uploaded File: KAMP_Settings.cfg
2025-01-20 10:15:58,250 [application.py:log_request()] - 201 POST /server/files/upload (192.168.1.14) [_TRUSTED_USER_] 15.48ms
2025-01-20 10:16:34,924 [application.py:prepare()] - Upload Request Received from 192.168.1.14
Content-Type: 
2025-01-20 10:16:34,934 [application.py:prepare()] - Upload Request Received from 192.168.1.14
Content-Type: multipart/form-data; boundary=----WebKitFormBoundaryMYVrhnbJhQqSSiSn
2025-01-20 10:16:34,965 [application.py:post()] - Processing Uploaded File: printer.cfg
2025-01-20 10:16:34,968 [application.py:log_request()] - 201 POST /server/files/upload (192.168.1.14) [_TRUSTED_USER_] 34.15ms
2025-01-20 10:17:58,746 [application.py:prepare()] - Upload Request Received from 192.168.1.14
Content-Type: 
2025-01-20 10:17:58,752 [application.py:prepare()] - Upload Request Received from 192.168.1.14
Content-Type: multipart/form-data; boundary=----WebKitFormBoundary8HzMUbxMBC9ZjZ2i
2025-01-20 10:17:58,758 [application.py:post()] - Processing Uploaded File: printer.cfg
2025-01-20 10:17:58,763 [application.py:log_request()] - 201 POST /server/files/upload (192.168.1.14) [_TRUSTED_USER_] 10.64ms
2025-01-20 10:17:59,817 [klippy_connection.py:close()] - Error closing Klippy Unix Socket
Traceback (most recent call last):
  File "/home/secretjameser/moonraker/moonraker/components/klippy_connection.py", line 802, in close
    await self.writer.wait_closed()
  File "/usr/lib/python3.11/asyncio/streams.py", line 350, in wait_closed
    await self._protocol._get_close_waiter(self)
  File "/usr/lib/python3.11/asyncio/selector_events.py", line 995, in _read_ready__data_received
    data = self._sock.recv(self.max_size)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ConnectionResetError: [Errno 104] Connection reset by peer
2025-01-20 10:17:59,817 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-01-20 10:17:59,818 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.info, Code: 503, Message: Klippy Disconnected
2025-01-20 10:17:59,818 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.objects.list, Code: 503, Message: Klippy Disconnected
2025-01-20 10:17:59,879 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.info, Code: 503, Message: Klippy Host not connected
2025-01-20 10:18:00,071 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-01-20 10:18:00,095 [machine.py:extract_service_info()] - Detected alternate unit name for klipper: klipper-Ender3Pro.service
2025-01-20 10:18:00,354 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-01-20 10:18:00,355 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-01-20 10:18:00,355 [klippy_connection.py:_check_ready()] - 
Include file '/home/secretjameser/Ender3Pro_data/config/sample-macros.cfg' does not exist

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted

2025-01-20 10:18:15,632 [application.py:prepare()] - Upload Request Received from 192.168.1.14
Content-Type: 
2025-01-20 10:18:15,641 [application.py:prepare()] - Upload Request Received from 192.168.1.14
Content-Type: multipart/form-data; boundary=----WebKitFormBoundarydBaCJiLAsftMWxOl
2025-01-20 10:18:15,645 [application.py:post()] - Processing Uploaded File: sample-macros (2).cfg
2025-01-20 10:18:15,647 [application.py:log_request()] - 201 POST /server/files/upload (192.168.1.14) [_TRUSTED_USER_] 6.05ms
2025-01-20 10:18:26,442 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2025-01-20 10:18:26,478 [common.py:build_error()] - JSON-RPC Request Error - Requested Method: printer.info, Code: 503, Message: Klippy Host not connected
2025-01-20 10:18:27,699 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2025-01-20 10:18:27,724 [machine.py:extract_service_info()] - Detected alternate unit name for klipper: klipper-Ender3Pro.service
2025-01-20 10:18:30,229 [klippy_connection.py:_request_initial_subscriptions()] - Webhooks Subscribed
2025-01-20 10:18:30,231 [klippy_connection.py:_request_initial_subscriptions()] - GCode Output Subscribed
2025-01-20 10:18:30,477 [job_state.py:_handle_started()] - Job state initialized: standby
2025-01-20 10:18:30,729 [klippy_connection.py:_check_ready()] - Klippy ready
2025-01-20 10:18:31,230 [data_store.py:_init_sensors()] - Configuring available sensors: ['heater_bed', 'extruder']
